### PR TITLE
fix: last acc hidden in switch acc list, closes #5975

### LINKED ISSUE #5975: Last account in the wallet when switching not fully visable
Image

### PR DESCRIPTION
**Pull request by alter-eggo**
> Try out Leather build 2945c8d — [Extension build](https://github.com/leather-io/extension/actions/runs/11973580745), [Test report](https://leather-io.github.io/playwright-reports/fix-switch-mb), [Storybook](https://fix-switch-mb--65982789c7e2278518f189e7.chromatic.com), [Chromatic](https://www.chromatic.com/library?appId=65982789c7e2278518f189e7&branch=fix-switch-mb)

This pr fixes last acc hidden in switch acc sheet and in choose acc flow

### CODE REVIEW

File: src/app/ui/components/virtuoso.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export function VirtuosoWrapper({ children, hasFooter, isPopup }: VirtuosoWrappe
   const [key, setKey] = useState(0);
   const isAtLeastMd = useViewportMinWidth('md');
   const virtualHeight = isAtLeastMd ? '70vh' : '100vh';
-  const headerHeight = isPopup ? 230 : 60;
+  const headerHeight = isPopup ? 300 : 80;
```
Comment (kyranjamie) [inline src/app/ui/components/virtuoso.tsx:23]: Seems like a bad choice to me that the core virtuoso component has these hard coded header/footer heights.

What if i use the component again with a different footer?

File: src/app/pages/choose-account/components/accounts.tsx [hunk 2]
```diff
@@ -85,7 +85,7 @@ function AddAccountAction() {
     <Box mx="space.05" py="space.02" onClick={onCreateAccount} {...bind}>
       <HStack alignItems="center">
         <PlusIcon />
-        Generate new account
+        <styled.span textStyle="label.02">Generate new account</styled.span>
       </HStack>
       {component}
```
Comment (kyranjamie) [inline src/app/pages/choose-account/components/accounts.tsx:90]: Shouldn't the container just be changed such that the height of the list stops when it gets to the button, and doesn't go under it?

**Review by kyranjamie — APPROVED** [review]
Approved as these decisions were prior to your PR, but some of these hard coded values don't look like the way to do it

File: src/app/ui/components/virtuoso.tsx (hunk 1 above)
Comment (alter-eggo) [reply src/app/ui/components/virtuoso.tsx:23]: I didn't want to start huge refactoring here

Comment (pete-watters) [reply src/app/ui/components/virtuoso.tsx:23]: This code isn't ideal but we have had this issue for a long time and this was the simplest fix. We only use this 3 times and twice with a footer. 

We could refactor to pass the values in but I think it's OK

**Review by pete-watters — APPROVED** [review]
Thanks for fixing this @alter-eggo 👍 

I'm almost certain it was fixed before so sad to see it re-appear but glad you sorted it

**Comment by alter-eggo** [comment]
close in favor of https://github.com/leather-io/extension/pull/5988

**Pull request closed without merging**